docs/src/guide/staking/index.md

```diff
@@ -0,0 +1,136 @@
+# {#guide-staking} Staking and Unstaking Tokens
+
+Reach makes DApp development safer, faster, and more secure.
+This guide offers a real example of the advantages of using Reach.
+
+Here's an example provided by Austin Wilshire at [xBacked](https://twitter.com/xbacked). 
+Follow Austin on Twitter [@awoldes](https://twitter.com/awoldes).
+
+Imagine you want to stake a token and at a later date unstake it with all the rewards.
+Let's look at a snippet from the perspective of a Deployer. 
+
+As the deployer, you want to set the reward rate, ASA ID of the token being stakes, the ASA ID of the rewarded token, and the initial supply of the reward token.
```

Comment:
We can't say "ASA ID". We'd have to say `{!rsh} Token` id

Reply:
Also, typo on this line, "staked"

Reply:
I recommend [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) if you're using VSCode to edit the Markdown, JP.